backend/packages/Upgrade/src/api/services/UserService.ts

```diff
@@ -67,6 +85,7 @@ export class UserService {
   }
 
   public updateUserRole(email: string, role: UserRole): Promise<User> {
+    this.sendRoleChangedEmail(email, role);
```

Comment:
Send the email after user role gets updated successfully in the database

Reply:
okay